back-end/influxdb.rb

```diff
@@ -0,0 +1,14 @@
+require 'influxdb'
+
+class Influxdb
```

Comment:
Can we call this class something like `InfluxClient` or `InfluxAdapter`?

Reply:
> Can we call this class something like `InfluxClient` or `InfluxAdapter`?

Renamed to InfluxClient